src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,152 @@
+$theme-focusHighlight: #1c64f2;
+$theme-primaryBackground: #1c325e;
+$theme-whiteColor: #ffffff;
+$theme-bannerFontColor: #f1416c;
+$theme-secondaryBackground: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-headingColor: #6b7280;
+$theme-inputFieldBorder: #d1d5db;
+$theme-nonActiveButtonBackground: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.88rem;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.standupBanner {
+    background-color: $theme-primaryBackground;
+    width: 84%;
+    height: 12%;
+    color: $theme-whiteColor;
+    padding: 2rem;
+    border-radius: $radius;
+}
+
+.standupBanner .bannerPara {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.24rem;
+    font-weight: 600;
+}
+
+.totalMissedUpdate {
+    color: $theme-bannerFontColor;
+    text-decoration: underline;
+    margin: 0 0.4rem;
+}
+
+.standupUpdateContainer {
+    width: 50.8%;
+    margin-top: 4rem;
+}
+
+.standupForm {
+    margin-top: 2.2rem;
+}
+
+.standupTitle {
+    color: $theme-secondaryBackground;
+    font-family: 'Inter', sans-serif;
+    font-size: 2.2rem;
```

Comment:
variable please

Reply:
I don't think we should use variables for font size because one font size is used on one element and is not repeating and as per my knowledge we create variables for those if those values are repeating or have chances of repeating to make our code look clean.
for example, border radius throughout the feature was 0.5rem so I created a variable for it and I am using it everywhere

Reply:
if we create a variable,lets say
title, content, subtitle, it would bring consistency across the product and help in changing the size around in a single place.

Reply:
suppose defined the size of a class using a variable and that class font size is different on different screen sizes so do you want me to define the different variable name for that also?
we are talking about clean code, do you think writing a variable name for each class will make the code look clean?